sdk/servicebus/service-bus/src/core/managementClient.ts

```diff
@@ -558,16 +512,10 @@ export class ManagementClient extends LinkEntity {
    */
   async cancelScheduledMessages(sequenceNumbers: Long[]): Promise<void> {
     throwErrorIfConnectionClosed(this._context.namespace);
-    if (!Array.isArray(sequenceNumbers)) {
```

Comment:
IIRC we had discussed keeping the Long type checks since it is a troublesome case where we do need a particular type passed in? We had discussed checking just for a `toBytesBE` method but rejected it since it's not really a generic extension point but particular to this library.

Reply:
We are doing the Long check. It has just been moved into the sender.ts class